grantnav/frontend/templates/components/grants_table_scripts.html

```diff
@@ -27,7 +33,7 @@
 const params = new URLSearchParams(window.location.search);
 const inputRows = params.get('rows');
 const inputColumns = params.getAll('widgetColumn');
-const isWidget = window.location.pathname.includes('widget');
+const isWidget = {% if widget %}true{% else %}false{% endif %};
```

Comment:
This looks good, but appears to break linting. Perhaps add a `// eslint-disable-line` comment to disable ESLint checking that line.